services/galley/test/integration/API/Federation.hs

```diff
@@ -73,45 +74,55 @@ tests s =
 
 getConversationsAllFound :: TestM ()
 getConversationsAllFound = do
-  -- FUTUREWORK: make alice / bob remote users
   [alice, bob] <- randomUsers 2
-  connectUsers alice (singleton bob)
-  -- create & get one2one conv
-  cnv1 <- responseJsonUnsafeWithMsg "conversation" <$> postO2OConv alice bob (Just "gossip1")
-  getConvs alice (Just $ Left [qUnqualified . cnvQualifiedId $ cnv1]) Nothing !!! do
-    const 200 === statusCode
-    const (Just [cnvQualifiedId cnv1]) === fmap (map cnvQualifiedId . convList) . responseJsonUnsafe
+  let aliceQ = Qualified alice (Domain "far-away.example.com")
```

Comment:
I don't fully understand what's going on. On line 77 you generate Alice as a random local user and here you associate her with a specific domain that might or might not be the local domain. Perhaps Alice should be generated with `randomId` instead?

Reply:
Yes, that was a mistake. Fixed in 77b62c13cf8022b87afcadc92de82d4154a7a352.